src/components/App/App.module.css

```diff
@@ -1,9 +1,13 @@
 .wrap {
     display: block;
     margin: auto;
-    width: 300px;
+    width: 500px;
+    background-color: #ffffff;
+    border-radius: 7px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693406)